zerver/openapi/zulip.yaml

```diff
@@ -9258,14 +9365,34 @@ paths:
                 items:
                   type: integer
               example: [9, 10]
-          required: true
+        - name: topic
+          in: query
+          description: |
+            Topic to which message is being typed. Valid only for type 'stream'.
```

Comment:
Should say "Required for the "stream" type" or something to that effect.

Reply:
The description now is:
>Topic to which message is being typed. Required for the 'stream' type.
            Ignored in case of 'private' type.